editor.planx.uk/src/pages/FlowEditor/lib/analytics/provider.tsx

```diff
@@ -40,8 +40,11 @@ export const ALLOW_LIST = [
   "application.declaration.connection",
   "drawBoundary.action",
   "findProperty.action",
+  "_overrides",
+  "planningConstraints.action",
   "property.constraints.planning",
   "property.type",
+  "propertyInformation.action",
```

Comment:
With this change we now have basic analytics coverage to understand how users proceed through our ALL our geospatial components (based on component type, not restricted to whether used in a submission or guidance flow!) :heavy_check_mark: